ci-operator/step-registry/openshift/e2e/libvirt/conf/openshift-e2e-libvirt-conf-commands.sh

```diff
@@ -311,8 +311,8 @@ fi
 #
 
 # Until the yellow-zone network bandwidth is upgraded, we will run the following tests in their own periodic and exclude
-# them from the conformance-parallel workflow.
-if [ "${TEST_TYPE}" == "conformance-parallel" ] || [ "${TEST_TYPE}" == "heavy-build" ]; then
+# them from the conformance-parallel workflow in versions older than 4.21.
+if echo ${BRANCH} | awk -F. '{ if (($1 == 4) && ($2 <= 20)) { exit 0 } else { exit 1 } }' &&  [[ "${TEST_TYPE}" == "conformance-parallel" || "${TEST_TYPE}" == "heavy-build" ]]; then
```

Comment:
basically excluding them if they are older than 4.20? who is going to remember? maybe it should be a variable that's updated per release?